hbase-server/src/test/java/org/apache/hadoop/hbase/replication/regionserver/TestRegionReplicaReplicationEndpoint.java

```diff
@@ -487,4 +488,25 @@ private void testRegionReplicaReplicationIgnores(boolean dropTable, boolean disa
       connection.close();
     }
   }
+  private void enableTableWithRetries(TableDescriptor htd, boolean createTableOperation) throws InterruptedException {
```

Comment:
The purpose of this method is to either `create` or `enable` table, so let's rename method name to `createOrEnableTableWithRetries`